common/grpc/protobuf-jackson/src/main/java/org/curioswitch/common/protobuf/json/SerializeSupport.java

```diff
@@ -0,0 +1,240 @@
+
+  static void printUnsignedInt64(long value, JsonGenerator gen) throws IOException {
+    gen.writeString(normalizeUnsignedInt64(value));
+  }
+
+  static String normalizeUnsignedInt64(long value) {
+    return value >= 0
+        ? Long.toString(value)
+        // Pull off the most-significant bit so that BigInteger doesn't think
+        // the number is negative, then set it again using setBit().
+        : BigInteger.valueOf(value & Long.MAX_VALUE).setBit(Long.SIZE - 1).toString();
+  }
+
+  static void printRepeatedBool(List<Boolean> values, JsonGenerator gen) throws IOException {
+    int numElements = values.size();
+    gen.writeStartArray(numElements);
+    for (int i = 0; i < numElements; i++) {
+      printBool(values.get(i), gen);
+    }
+    gen.writeEndArray();
+  }
+
+  static void printBool(boolean value, JsonGenerator gen) throws IOException {
+    gen.writeBoolean(value);
+  }
+
+  static void printRepeatedFloat(List<Float> values, JsonGenerator gen) throws IOException {
+    int numElements = values.size();
+    gen.writeStartArray(numElements);
+    for (int i = 0; i < numElements; i++) {
+      printFloat(values.get(i), gen);
+    }
+    gen.writeEndArray();
+  }
+
+  static void printFloat(float value, JsonGenerator gen) throws IOException {
+    gen.writeNumber(value);
+  }
+
+  static void printRepeatedDouble(List<Double> values, JsonGenerator gen) throws IOException {
+    int numElements = values.size();
+    gen.writeStartArray(numElements);
+    for (int i = 0; i < numElements; i++) {
+      printDouble(values.get(i), gen);
+    }
+    gen.writeEndArray();
+  }
+
+  static void printDouble(double value, JsonGenerator gen) throws IOException {
+    gen.writeNumber(value);
+  }
+
+  static void printRepeatedString(List<String> values, JsonGenerator gen) throws IOException {
+    int numElements = values.size();
+    gen.writeStartArray(numElements);
+    for (int i = 0; i < numElements; i++) {
+      printString(values.get(i), gen);
+    }
+    gen.writeEndArray();
+  }
+
+  static void printString(String value, JsonGenerator gen) throws IOException {
+    gen.writeString(value);
+  }
+
+  static void printRepeatedBytes(List<ByteString> values, JsonGenerator gen) throws IOException {
+    int numElements = values.size();
+    gen.writeStartArray(numElements);
+    for (int i = 0; i < numElements; i++) {
+      printBytes(values.get(i), gen);
+    }
+    gen.writeEndArray();
+  }
+
+  static void printBytes(ByteString value, JsonGenerator gen) throws IOException {
+    gen.writeBinary(value.toByteArray());
+  }
+
+  static void printEnum(EnumValueDescriptor value, JsonGenerator gen) throws IOException {
+    if (value.getIndex() == -1) {
+      gen.writeString(Integer.toString(value.getNumber()));
+    } else {
+      gen.writeString(value.getName());
+    }
+  }
+
+  // Note: I hope no one ever actually calls this method...
+  static void printRepeatedNull(List<Integer> values, JsonGenerator gen) throws IOException {
```

Comment:
null is always represented as an enum in proto, which we work with using int. I want to keep it consistent with printRepeatedEnum